core/common/src/PlatformUtilities.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { BentleyStatus, IModelError } from "./IModelError";
+import type { Readable, Writable } from "stream"; // Must be "import type" to avoid polyfill errors
+import type { Buffer } from "buffer"; // Must be "import type" to avoid polyfill errors
```

Comment:
what does "polyfill errors" mean?

Reply:
Webpack no longer automatically polyfills node types like Buffer anymore. So, if we generate code here (import instead of import type), webpacks would fail.